tools/actions_local_runner.py

```diff
@@ -70,65 +78,169 @@ def find_changed_files() -> List[str]:
     return [x.strip() for x in all_files if x.strip() != ""]
 
 
-async def run_step(step: Dict[str, Any], job_name: str, files: Optional[List[str]], quiet: bool) -> bool:
+def print_results(job_name: str, passed: bool, streams: List[str]) -> None:
+    header(job_name, passed)
+    for stream in streams:
+        stream = stream.strip()
+        if stream != "":
+            print(stream)
+
+
+async def shell_cmd(
+    cmd: str, env: Optional[Dict[str, Any]] = None, redirect: bool = True
+) -> Tuple[bool, str, str]:
+    proc = await asyncio.create_subprocess_shell(
+        cmd,
+        shell=True,
+        cwd=REPO_ROOT,
+        env=env,
+        stdout=subprocess.PIPE if redirect else None,
+        stderr=subprocess.PIPE if redirect else None,
+        executable=shutil.which("bash"),
+    )
+    stdout, stderr = await proc.communicate()
+
+    passed = proc.returncode == 0
+    if not redirect:
+        return passed, "", ""
+
+    return passed, stdout.decode(), stderr.decode()
+
+
+def header(name: str, passed: bool) -> None:
+    PASS = color(col.GREEN, "\N{check mark}")
```

Comment:
whoa, I didn't know this was a thing! but, might it be more straightforward to just add
```py
# -*- coding: utf-8 -*-
```
as the second line of the file, and then change this line to just be the following?
```suggestion
    PASS = color(col.GREEN, "✓")
```
totally up to you though